options/options.html

```
@@ -822,84 +829,128 @@ <h3>
-              </label>
-              <p class="help-text">關閉後不會自動顯示右側小按鈕，但仍可透過 Popup 開始標註。</p>
+              <p
+                class="help-text"
+                data-ui-message="OPTIONS.TEMPLATES.HIGHLIGHT_CONTENT_STYLE_HELP"
+              ></p>
             </div>
           </div>
 
-          <button id="save-templates-button" class="btn-primary full-width">儲存模板設定</button>
+          <button
+            id="save-templates-button"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)